lib/double_entry/reporting.rb

```diff
@@ -78,8 +78,8 @@ class AggregateFunctionNotSupported < RuntimeError; end
     # @raise [Reporting::AggregateFunctionNotSupported] The provided function
     #   is not supported.
     #
-    def aggregate(function, account, code, range, options = {})
-      Aggregate.formatted_amount(function, account, code, range, options)
+    def aggregate(function, account, code, range, filter: nil, partner_account: nil)
```

Comment:
I've opted not to at this time.   It doesn't seem worth it to stop supporting Ruby 2.0.0 just for required keyword arguments (since it's still a supported ruby version)